### ISSUE TITLE
'ziti/ziti.h' file not found

### ISSUE THREAD

**Issue by McGonagall666** [openziti/ziti-sdk-swift#206]
Hello, where is the Ziti library? How can I use the things in lib?For example, ZitiEnroller（Cannot find type 'uv_loop_t' in scope）. I feel like there is a lack of key libraries. Please let me know. Thank you very much

**Comment by scareything** [comment]
Hi. `ziti-sdk-swift` uses https://github.com/openziti/ziti-sdk-c, which is actually pulled in as a transitive dependency through https://github.com/openziti/ziti-tunnel-sdk-c. ziti-tunnel-sdk-c is referenced by this project as a git submodule in the deps/ subdirectory. Most of this info is in the [build instructions](https://github.com/openziti/ziti-sdk-swift/blob/main/README.md#execute-build-script), but here's a little more detail in case it helps:

1. You need to get submodules of the `ziti-sdk-swift` repo. You can do this when cloning with the ` --recurse-submodules` option. If you've already cloned then you can run `git submodule update --recursive` to get them.
2. Once your deps/ subdirectory contains ziti-tunnel-sdk-c you can build it using `build_all.sh` in this repo. Be sure to read the [requirements section in build instructions](https://github.com/openziti/ziti-tunnel-sdk-c/blob/main/BUILD.md#building-requirements). Specifically you'll need to have cmake and ninja installed (I use homebrew for these), and the `VCPKG_ROOT` environment variable exported and set to the path of a fetched and bootstrapped (built) clone of https://github.com/microsoft/vcpkg. (If you're running on Apple Silicon then you'll need to export and`VCPKG_FORCE_SYSTEM_BINARIES` to any value to satisfy vcpkg.)

**Comment by McGonagall666** [comment]
Thank you very much for your help. I have successfully started the project. 
I have reviewed the Building section below in detail and listened to your suggestions. I have installed cmake, ninja, vcpkg, and pkg-config respectively, and modified the VCPKG_ROOT in the files under deps.  
I'm M1's computer, but I couldn't find VCPKG_ FORCE_ SYSTEM_BINARIES Where is set, it has not been modified.
VCPKG_ROOT have a problem. After installing it, I couldn't modify ENV and couldn't find the corresponding cmake, so I added the information about VCPKG_ROOT in the file uses the absolute path of files under vcpkg, and then goes to/bin/sh build_ all.sh, finally compiled successfully.
/bin/sh build_all.sh Many times during, I encountered Failed to connect to github.com port 443. Just turn off the agent and wait for a long time. Don't worry, I spent the whole morning, and finally succeeded
Thank you,
ZHANGFENG
